core/vm/evm.go

```diff
@@ -202,6 +202,7 @@ func (evm *EVM) Call(caller ContractRef, addr common.Address, input []byte, gas
 
 	if !evm.StateDB.Exist(addr) {
 		if !isPrecompile && evm.chainRules.IsEIP158 && value.Sign() == 0 {
+			// TODO proof of absence here?
```

Comment:
yes

Reply:
touchAddressAndChargeGas(addr)

Reply:
I think it's important that we distinguish between between the case where we want a proof of absence, and the case where we are waiting for another call to set the value.  I can't think of an example where this could break at this moment.  But I have a feeling it can and will try to think of a way to do it.

Reply:
but for the sake of getting this PR merged, I'll just add a call to `TouchAddressAndChargeGas(addr, nil)` here.